app/views/donations/index.html.erb

```diff
@@ -1,13 +1,48 @@
-<div align="center">
-<div class="container-fluid" style="padding-top: 20px; padding-left:; width: 30%; float: centered;">
-	<h1 class="container-fluid"> Support Our Cause </h1>
+
+
+<div align="center" style= "padding-top: 20px; border: solid; background: #129EA2; color: #129EA2; border-width: .5px;" >
+	<div class="top-right"  >
+		<div class="container-fluid" style="margin-top: 3%; padding-bottom: 45px; width: 30%; float: right; margin-right: 20%">
+			<%= image_tag("https://scontent-sjc3-1.xx.fbcdn.net/v/t31.0-8/10355612_1466423220317340_2509667441501223212_o.png?_nc_cat=109&_nc_ht=scontent-sjc3-1.xx&oh=a0a3aa854e49d3c92d30d3063a42bc74&oe=5D6057B5", :alt => "rss feed", align: "center", style: "float: centered; margin-bottom: 10px; width: 70%; height 70%;") %>
```

Comment:
You should download the image and link to it locally rather than wherever this is. There's no guarantee this link will always be working.